src/components/custom/_signup-new.js

```diff
@@ -193,13 +192,19 @@ const SignupNew = ({
 }) => {
     const [checkBoxState, setCheckBoxState] = useState(false)
 
+    // Correct
+    const [key, setKey] = useState(undefined)
```

Comment:
```suggestion
    const [language_code, setLanguageCode] = useState(undefined)
```

Reply:
Nit: I think calling it `language_code` will be more descriptive than `key`. Should we also set the initial language code to english instead of undefined?

Reply:
done and thanks:)